docs/configuration.md

```diff
@@ -31,15 +63,13 @@ where
 - `{hostname}` is tge server name, from the `$HOST` or `$HOSTNAME` environment variable or `os.hostname()`
 - `{deployment}` is the deployment name, from the `$NODE_ENV` environment variable
 
-## SuiteC Configuration Options
-
-SuiteC allows the following values to be configured:
+### Customizations
 
-### `app` configuration
+#### `app` configuration
 
 - `port`: The internal port on which the SuiteC app server should listen
```

Comment:
@johncrossman Might be worth mentioning the `https` and `host` options here also. `host` should be set to `localhost` for local deployments.